### ISSUE TITLE
Getting AttributeError: token_type_hint

### ISSUE THREAD

**Issue by Gr1N** [idan/oauthlib#365]
Hi! 

I'm getting attribute error when trying to revoke `access_token` without `token_type_hint` parameter.
I think that the problem is the same as in the issue #364. Breaking commit: https://github.com/idan/oauthlib/commit/539558a02edbadb8d6ef690fb0beb2510a5eeb17.

And here a part of stack trace:
```
  File "./venv/lib/python2.7/site-packages/oauthlib/oauth2/rfc6749/endpoints/base.py", line 64, in wrapper
    return f(endpoint, uri, *args, **kwargs)
  File "./venv/lib/python2.7/site-packages/oauthlib/oauth2/rfc6749/endpoints/revocation.py", line 65, in create_revocation_response
    self.validate_revocation_request(request)
  File "./venv/lib/python2.7/site-packages/oauthlib/oauth2/rfc6749/endpoints/revocation.py", line 127, in validate_revocation_request
    if (request.token_type_hint and
  File "./venv/lib/python2.7/site-packages/oauthlib/common.py", line 409, in __getattr__
    raise AttributeError(name)
AttributeError: token_type_hint
```

**Comment by masci** [comment]
Yep, it's the same problem, a fix is already in place waiting for review in #362

**Comment by Gr1N** [comment]
Nice!

**Comment by masci** [comment]
Fixed in #362
